src/iuse.cpp

```diff
@@ -625,175 +625,111 @@ int iuse::atomic_caff(player *p, item *it, bool, const tripoint& )
     return it->type->charges_to_use();
 }
 
-int iuse::raw_meat(player *p, item *it, bool, const tripoint& )
+int raw_food(player *p, item *it, int c_tp, int c_bl, int c_br, int c_pc)
```

Comment:
I think I see what you're doing, c_tp == "chance of tapeworm"?
If so please expand them to "tapeworm_chance" or similar to make it a little obvious what's happening.
Another nice way to go would be to do something like:
```
struct parasite_probability {
    int tapeworm;
    int bloodworms;
    int brainworms;
    int paincysts;
};
```
and invoking it would look like:
```
int iuse::raw_meat(player *p, item *it, bool, const tripoint& )
{
    struct parasite_probability parasites;
    parasites.tapeworm = 32;
    parasites.bloodworms = 64;
    parasites.brainworms = 128;
    parasites.paincysts = 64;
    return raw_food(p, it, parasites);
}
```
and raw_food would look like:
```
int raw_food(player *p, item *it, const struct parasite_probability &parasites ) {
    ...
    if( parasites.tapeworm && one_in(parasites.tapeworm) && ...
}
```
It's roughly as verbose, but everything is packaged up and labelled.